src/common/components/Table.js

```diff
@@ -1,18 +1,73 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
+import _ from 'lodash'
 import { DataGrid } from '@mui/x-data-grid'
 
 const PAGE_SIZE = 15
+const DIRECTION_BY_WORD = {
```

Comment:
nitpick: I would recommend using `order` instead of `direction` here.

Reply:
@cacarrara I am going to change them to `SORT_DIRECTION_BY_WORD` and `SORT_DIRECTION_BY_SYMBOL` because it is the direction related to sort, I think this is clearer